src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -544,8 +544,10 @@ public function testMessengerAdapter()
         $container = $this->createContainerFromFile('messenger_adapter');
         $this->assertTrue($container->hasDefinition('messenger.sender.default'));
         $this->assertTrue($container->getDefinition('messenger.sender.default')->hasTag('messenger.sender'));
+        $this->assertEquals(array(array('name' => 'default')), $container->getDefinition('messenger.sender.default')->getTag('messenger.sender'));
```

Comment:
assertSame (same evrywhere)

Reply:
Updated for the valid ones (not for the ones containing `Reference`s...).